tasks/src/text-to-image/data.ts

```diff
@@ -0,0 +1,34 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+    datasets: [
+        {
+            description: "A large-scale image dataset containing images of everyday objects and people with their textual descriptions.",
+            id: "ChristophSchuhmann/MS_COCO_2017_URL_TEXT",
+        },
+        {
+            description: "A dataset created from Midjourney User Prompts & Generated Images",
+            id: "nateraw/midjourney-texttoimage-new",
+        },
+    ],
+    demo: {
```

Comment:
There's indentation and space related syntax errors in your code, you can use eslint if you want a quick fix, this is how it should look (you can see an example below) 🙂 
<img width="474" alt="lint" src="https://user-images.githubusercontent.com/53175384/194856290-1b947bcc-67be-4463-a7a1-db320fae67ad.png">

Reply:
Thank you for letting me know for the indent errors. I have committed the fixed version.